.circleci/config.yml

```diff
@@ -3,12 +3,12 @@
 
 references:
 
-  container_config_node8: &container_config_node8
+  container_config_node12: &container_config_node12
     working_directory: ~/project/build
     docker:
-      - image: circleci/node:8-browsers
+      - image: circleci/node:12-browsers
 
-  container_config_lambda_node8: &container_config_lambda_node8
+  container_config_lambda_node12: &container_config_lambda_node12
     working_directory: ~/project/build
     docker:
       - image: lambci/lambda:build-nodejs8.10
```

Comment:
**Chore:** We should update the version number here too.

Reply:
Cheers!